src/components/selection.ts

```diff
@@ -353,24 +353,47 @@ export default class SelectionUtils {
   }
 
   /**
-   * Adds fake cursor to the current range
+   * Check if current range exists and belongs to container
    *
-   * @param [container] - if passed cursor will be added only if container contains current range
+   * @param container - where range should be
    */
-  public static addFakeCursor(container?: HTMLElement): void {
+  public static isRangeInsideContainer(container: HTMLElement): boolean {
     const range = SelectionUtils.range;
-    const fakeCursor = $.make('span', 'codex-editor__fake-cursor');
 
-    fakeCursor.dataset.mutationFree = 'true';
+    if (range === null) {
+      return false;
+    }
+
+    return container.contains(range.startContainer);
+  }
+
+  /**
+   * Adds fake cursor to the current range
+   */
+  public static addFakeCursor(): void {
+    const range = SelectionUtils.range;
 
-    if (!range || (container && !container.contains(range.startContainer))) {
+    if (range === null) {
       return;
     }
 
+    const fakeCursor = $.make('span', 'codex-editor__fake-cursor');
+
+    fakeCursor.dataset.mutationFree = 'true';
+
     range.collapse();
     range.insertNode(fakeCursor);
   }
 
+  /**
+   * Check if passed element contains a fake cursor
+   *
+   * @param el - where to check
+   */
+  public static isElementContainsFakeCursor(el: HTMLElement): boolean {
```

Comment:
Not sure it's better but grammatically correct

```suggestion
  public static doesElementContainFakeCursor(el: HTMLElement): boolean {
```

Reply:
renamed with `isFakeCursorInsideContainer` to make it consistent with `isRangeInsideContainer`